.gitignore

```diff
@@ -0,0 +1,7 @@
+.settings/
+target/
+.project
+.classpath
```

Comment:
I would move those two (`.project` & `.classpath`) into your global gitignore, because github doesn't need to know which IDE you use, if that makes sense